crates/core/src/backend/ignore.rs

```diff
@@ -472,6 +472,41 @@ fn get_group_by_gid(gid: u32) -> Option<String> {
     }
 }
 
+#[cfg(target_os = "openbsd")]
```

Comment:
I think `#[cfg(all(not(windows),target_os = "openbsd"))]`should be used here

Reply:
yeah, I forgot to backport the (caller) `map_entry` `cfg`  to the functions I extracted.

This explains the cross compilation error.
[66c1aed](https://github.com/rustic-rs/rustic_core/pull/233/commits/66c1aed810214b27924c8a69477dbbcd1472b0c1) should solve this